src/pages/PlaylistManagementPage/PlaylistManagementPage.js

```diff
@@ -32,23 +33,45 @@ const PlaylistManagementPage = (props) => {
   }, [])
 
   const playlistClickedHandler = (playlistId) => {
+    const playlists = [...collaborativePlaylists]
+    const playlist = playlists.find((playlist) => playlist.id === playlistId)
+
+    if(playlist.selected) {
+      axios
+        .delete(
+          `${process.env.REACT_APP_API_BASE_URL}/playlist/${playlistId}`,
+          { withCredentials: true },
+        )
+        .then((response) => {
+          playlist.selected = !playlist.selected
```

Comment:
isn't it better to use 'false' to clearly define that we want the button not to be selected (increases readability, I think)?

Reply:
This will just flip the selected state of the current playlist.